src/test/lib/shady-render_test.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {html, render} from '../../lib/shady-render.js';
+
+
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
+/// <reference path="../../node_modules/@types/chai/index.d.ts" />
+
+const assert = chai.assert;
+
+suite('shady-render', () => {
+
+  test('prepares templates with ShadyCSS', () => {
+    const container = document.createElement('div');
+    render(html`
+      <style>
```

Comment:
Let's add a test that includes multiple style elements and one that uses nested templates.

Reply:
Done. It seems to actually do the right thing in Chrome at least.